src/index.js

```diff
@@ -293,3 +293,9 @@ module.exports = {
   }
 };
 
+var OutboundLink = require('./components/OutboundLink');
+OutboundLink.trackLink = reactGA.outboundLink;
```

Comment:
I would like to defer this to @adamlofting since he knows the usage convention and Google GA API the best :wink: 

From my understanding we are using the `ga.event` to track events and links within the site domain ([example](https://github.com/mozilla/teach.webmaker.org/issues/1109#issuecomment-123222969)) whereas [`ga.outboundLink`](https://github.com/adamlofting/react-ga/#gaoutboundlinkargs-hitcallback) is designed to track external URLs